### DISCUSSION TITLE
c++ iostream header not found

### DISCUSSION THREAD

**Discussion by Kim-Robin** [LazyVim/LazyVim#2271] [Q&A] [answered]
![Screenshot from 2023-12-29 10-55-52](https://github.com/LazyVim/LazyVim/assets/41379739/f47a1483-4b86-427d-9f68-6df845dc3a49)

I am able to run on terminal with g++ but I am not able to make lazyvim to recognize it. 

I am using lazy extra clangd for it.
{ import = "lazyvim.plugins.extras.lang.clangd" }

**Comment by dpetka2001** [comment, marked as answer]
Have you read the clangd [docs](https://clangd.llvm.org/guides/system-headers#headers-not-present-in-the-system-at-all)? Maybe that's what you're missing? Maybe try to install `libstdc++-12-dev` if you're on debian/ubuntu like system?

**Reply by Kim-Robin** [reply]
Thank you. It is working
